templates/docs/examples/patterns/image/spacing.html

```diff
@@ -0,0 +1,13 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image / Spacing{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
```

Comment:
Once we have a new SCSS file for this component by itself, a new standalone stylesheet will also need to be created inside `/scss/standalone` folder, that should only include styles for this component (not whole Vanilla).
In our case it would be media container and rule component (because we use the rule in the example).

I can give more context on that if needed, just let me know.